packages/material-icon/README.md

```diff
@@ -22,6 +22,16 @@ with CSS:
 import '@material/react-material-icon/dist/material-icon.css';
 ```
 
+#### Importing Icons
+
+The React material icon package does not come packaged with [Google Font's Material Icons](https://google.github.io/material-design-icons/). If it was included in the Sass package this would block rendering of the page until the icons download.
+We recommend following the two methods of adding the Material Icons to your app documented [here](https://google.github.io/material-design-icons/#getting-icons). We have personally taken the `<link>` tag approach as it doesn't block page render. Code is pasted here for your convenience.
```

Comment:
"following the two methods" ... this is unclear. Do you mean the developer has to do two things? Or the developer should pick one way or another. And which one is the `<link>` approach?

Reply:
There are 2 ways that are documented within on material icons. How about...
```
We recommend following either of the 2 ways of adding Material Icons to your app, which are both documented ....

The 1st documented solution is the route our team has taken, since its not bundled with our CSS. This allows for quicker rendering, but a possibility of a Flash Of Unstyled Content (**FOUC**).
```